kubetemplates/ingress.yml

```diff
@@ -21,20 +21,27 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
     nginx.ingress.kubernetes.io/proxy-body-size: 10m
     nginx.ingress.kubernetes.io/proxy-send-timeout: "360"
-    
+    nginx.ingress.kubernetes.io/use-regex: "true"
 #    ingress.kubernetes.io/whitelist-source-range: '128.112.0.0/16 140.180.0.0/16 63.237.233.8/32 73.97.166.124/30 24.56.235.125/32'
 spec:
   rules:
   - host: ${DOMAIN_NAME}
     http: &http_rules
       paths:
-      - path: /segmentation
+      - path: /segmentation/.*/(split|merge|split_preview)$
```

Comment:
Can't `split_preview` be part of the read-services? Conversely, `undo`, `redo` and `rollback` all involve write operations - do you need to add those here?

Reply:
yes good catch

Reply:
I think although you are right about split_preview it has memory requirements that are as high as split and so we want it on the beefier nodes.

Reply:
this is addressed

Reply:
Oh, I thought the difference is just the preemption mode, but all calls would be handled by faster n2d/t2d instances? You want to keep the read-only calls on the e2 instances?